pkg/influx/parser.go

```diff
@@ -73,7 +73,7 @@ func influxPointToTimeseries(pt models.Point) ([]cortexpb.TimeSeries, error) {
 
 	fields, err := pt.Fields()
 	if err != nil {
-		return nil, fmt.Errorf("error getting fields from point: %w", err)
+		return nil, errorx.Internal{Msg: "error getting fields from point", Err: err}
```

Comment:
I am not exactly sure what kind of error this is, but it seems like an internal, not a client error.